signac/contrib/job.py

```diff
@@ -528,7 +537,7 @@ def doc(self):
 
         Returns
         -------
-        :class:`~signac.synced_collections.backends.collection_json.BufferedJSONDict`
+        :class:`~signac.synced_collections.backends.collection_json.BufferedJSONAttrDict`
```

Comment:
If we import the `BufferedJSONAttrDict as JSONDict` above, then we should revert to the old name `signac.JSONDict`.